src/TestUtils/src/DeviceTests/AssertionExtensions.iOS.cs

```diff
@@ -528,5 +529,50 @@ public static UIView GetAccessiblePlatformView(this UIView platformView)
 
 			return platformView;
 		}
+
+		public static void TapBackButton(this UINavigationBar uINavigationBar)
```

Comment:
```suggestion
		public static void GetTapBackButton(this UINavigationBar uINavigationBar)
```

would this be accurate?